### DISCUSSION TITLE
Configuration process for bcrypt, blowfish, and passhash9

### DISCUSSION THREAD

**Discussion by jthompsonic** [randombit/botan#3257] [Q&A] [answered]
The passhash methods + blowfish seem to have been left out of the configuration process.  Was this intentional? If not I could add a pull request to add them into configure.py as a passhash module option (for bcrypt and passhash9) ... I would add blowfish as a block cipher option.

**Comment by randombit** [comment, marked as answer]
Hi - can you give a bit more context here? The password hashing mechanisms are certainly included when I configure locally and I can see them being built on CI as well. Or do you mean, that it is possible for `passhash9` to be built without Blowfish being included? This is intentional, since for policy reasons some users disable Blowfish as it is not an allowed block cipher, but otherwise would be allowed to use passhash9 since it is based on the (approved) PBKDF2, and SHA-2 is an alternative option.

**Reply by jthompsonic** [reply]
Thank you.  I'm a newbie on building botan and I missed how how to identify the modules I was interested in (bcrypt, blowfish and passhash9) by doing the: configure.py --list-modules and then adding them to my configure.py --enable-modules=<modules-list-to-enable>  so my  quandary is resolved. 

Thanks for such a great understandable and logical library with associated program examples.
